src/v1/sources/shopify/transform.js

```diff
@@ -1,9 +1,11 @@
 /* eslint-disable @typescript-eslint/naming-convention */
-const { processEventFromPixel } = require('./pixelTransform');
+const { processEventFromPixel } = require('./webpixelTransformations/pixelTransform');
 const { process: processWebhookEvents } = require('../../../v0/sources/shopify/transform');
+const { process: processWebhookEventsV2 } = require('./webhookTransformations/serverSideTransform');
 
 const process = async (inputEvent) => {
   const { event } = inputEvent;
+  const { query_parameters } = event;
   // check on the source Config to identify the event is from the tracker-based (legacy)
```

Comment:
remove this comment - no longer valid
```suggestion
  // check on the source Config to identify the event is from the tracker-based (legacy)
```

Reply:
thanks, updated it to describe the current flow